### ISSUE TITLE
QA: Check if steam is installed in wineprefix before installation

### ISSUE THREAD

**Issue by Kreyren** [PhoenicisOrg/scripts#998]
I was trying to install Mafia II where it's installation failed, but steam was installed. While i try to run the same installation again it prompts me to install Steam which is already present in wineprefix which is not expected.

Recommends adding a sanity check for steam verb to check if steam is installed in said wineprefix to avoid duplicated. Same logic should also be included in other verbs.

**Comment by ImperatorS79** [comment]
I had no problem installing a steam apps today. However, It seems that the downloader "Downloading runtime json..." message is displayed everytime, the order messages only appear a few milliseconds. @plata any idea why ?

**Comment by plata** [comment]
The message should be printed every time according to the code:
https://github.com/PhoenicisOrg/scripts/blob/53a6846f18c9841225d063a1a5d15c7aa94abbdd/Engines/Wine/Engine/Implementation/script.js#L112
https://github.com/PhoenicisOrg/scripts/blob/53a6846f18c9841225d063a1a5d15c7aa94abbdd/Engines/Wine/Engine/Implementation/script.js#L127

**Comment by ImperatorS79** [comment]
I though it would be printed only when the file is actually being downloaded.

**Comment by plata** [comment]
The json is downloaded every time. The runtime itself is downloaded only if the json is different.

**Comment by ImperatorS79** [comment]
Yes, but when the download is finished and a new message occurs, the json downloading message should disappear. But is seems it is displayed over and over while the installation process, like a default message.

**Comment by plata** [comment]
Which message do you expect instead?

**Comment by ImperatorS79** [comment]
All the message in SteamScript.go() for example.

**Comment by Kreyren** [comment]
@ImperatorS79 which are? assuming it beeing wine output  i doubt that phoenicis windows is optimize to handle them.. 

Also stop fragmenting issues this issue is about sanitization of steam script.

**Comment by ImperatorS79** [comment]
This is not any wine output, I am talking about phoenicis message that guides you during the installation process. But you are right, I should open a new issue for that.
